pkg/pex/option.go

```diff
@@ -91,22 +98,31 @@ func WithTick(d time.Duration) Option {
 //
 // Users SHOULD ensure all nodes in a given cluster have
 // the same maximum view size.
-func WithMaxViewSize(n uint) Option {
-	if n == 0 {
-		n = 32
+func WithGossipParams(gossip GossipParams) Option {
```

Comment:
Fields of `GossipParams` are not exported, so users cannot parametrize gossip params.  This might be a good thing, in which case we shouldn't export this option.  If we do want to give this power to users, however, we'll need to export the fields of `GossipParams`.

Reply:
Okay, yes, we will export them.